dev/tools/svg2dart/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: svg2dart
+description: A tool for generating Dart vector animation code from SVG sequences.
+version: 0.0.1
+homepage: https://flutter.io
+author: Flutter Authors <flutter-dev@googlegroups.com>
+
+environment:
+  sdk: '>=1.20.1 <2.0.0'
+
+dependencies:
+  args: ^1.2.0
+
+dev_dependencies:
+  test: ^0.12.0
```

Comment:
you need to run `flutter update-packages --force-upgrade` just before you check this in (maybe first run it in a separate branch so that it upgrades everything else, then run it for you)

Reply:
I'll do that before submitting.